src/Generator.h

```diff
@@ -2304,10 +2304,18 @@ class GeneratorStub : public NamesInterface {
         return get_first_output().realize(std::forward<Args>(args)..., get_target());
     }
 
-    template<typename Dst>
-    void realize(Dst dst) {
+    template<typename Realization>
+    void realize(Realization r) {
         check_scheduled("realize");
-        get_first_output().realize(dst, get_target());
+        get_first_output().realize(r, get_target());
+    }
+
+    // Note that the Realization here is expected to have the appropriate number-and-type
+    // of buffers to realize *all* of this Generator's Outputs (not just the first one).
```

Comment:
The other realize() wrappers in GeneratorStub apply only to the first Output; was seeking to avoid confusion. Looks like I also caused it.